streamlit_web.py

```diff
@@ -357,6 +358,10 @@ def question_freq(freq_range):
 def sentiment():
     """main function for sentiment analysis"""
     senti_df = main_df.copy(deep=True)
+    # Initializing the new columns with a numpy array, so the entire series is returned
+    senti_df[cts.POSITIVE] = az.senti(senti_df[cts.TOKEN].values, 1)
+    senti_df[cts.NEGATIVE] = az.senti(senti_df[cts.TOKEN].values, -1)
```

Comment:
I can't help but notice that this `senti` function is being called twice, meaning that `create_word_list` (the unique words, the sentiment call, the sort, etc.) is called twice. Since there was a performance concern of this PR, I think it would be best if one could come up with a solution that's less redundant.

Perhaps save the list created by `create_word_list` in the data frame and just select the positive and negative words; or perhaps return both the `positive` and `negative` word list in one function call. That should at least save half of the time.